src/jarabe/desktop/keydialog.py

```diff
@@ -127,7 +127,7 @@ def create_security(self):
     def get_response_object(self):
         return self._response
 
-    def __toggle_visibility_cb(self, entry):
+    def _toggle_visibility_cb(self, entry):
```

Comment:
It seems to be the convention to use a double underscore for callbacks.  Can you change the caller instead?